packages/app-bridge/src/registries/api/ApiMethodRegistry.ts

```diff
@@ -1,10 +1,15 @@
 /* (c) Copyright Frontify Ltd., all rights reserved. */
 
 import type { ApiMethodNameValidator } from '../../AppBridge';
+import type { GetAssetBulkDownloadTokenPayload, GetAssetBulkDownloadTokenResponse } from './GetAssetBulkDownloadToken';
 import type { GetCurrentUserPayload, GetCurrentUserResponse } from './GetCurrentUser';
 import type { GetAssetResourceInfoPayload, GetAssetResourceInfoResponse } from './GetAssetResourceInfo';
 
 export type ApiMethodRegistry = ApiMethodNameValidator<{
+    getAssetBulkDownloadToken: {
+        payload: GetAssetBulkDownloadTokenPayload;
+        response: GetAssetBulkDownloadTokenResponse;
+    };
```

Comment:
Now we share the apiMethods for blocks and platformApps. What do you think, maybe we should separate these, otherwise we have method available in the block that are not implemented.

Reply:
We just share the registry. We guard around the AppBridge with the `PlatformAppApiMethod` and `BlockApiMethod`. So I don't think we need to split it. 
Do you see something I don't see?

Reply:
ah then perfect 👍